.github/workflows/build-images-ci.yaml

```diff
@@ -90,6 +87,21 @@ jobs:
           mkdir -p ../cilium-base-branch
           cp -r .github/actions/set-runtime-image ../cilium-base-branch
 
+      - name: Check for disk usage and cleanup /mnt
+        shell: bash
+        run: |
+          df -h
+          sudo rm -f '/mnt/tmp-pv.img'
```

Comment:
Any thoughts on whether to integrate this into `./.github/actions/disk-cleanup` ? If this fails, we'll lose visibility into the disk usage. This approach is also skipping any other file cleanup steps (which we may rely on for some runners).